src/main/java/com/exadel/sandbox/officeFloorAttachment/controllers/service/impl/AttachmentServiceImpl.java

```diff
@@ -0,0 +1,75 @@
+package com.exadel.sandbox.officeFloorAttachment.controllers.service.impl;
+
+import com.exadel.sandbox.officeFloorAttachment.controllers.dto.AttachmentDto;
+import com.exadel.sandbox.officeFloorAttachment.controllers.entities.Attachment;
+import com.exadel.sandbox.officeFloorAttachment.controllers.exception.EntityNotFoundException;
+import com.exadel.sandbox.officeFloorAttachment.controllers.repositories.AttachmentRepository;
+import com.exadel.sandbox.officeFloorAttachment.controllers.service.AttachmentService;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Component
+@RequiredArgsConstructor
+public class AttachmentServiceImpl implements AttachmentService {
+
+    private final ModelMapper mapper;
+
+    @Autowired
+    AttachmentRepository attachmentRepository;
+
+    @Override
+    public List<AttachmentDto> getAttachments() {
+        List<Attachment> all = attachmentRepository.findAll();
```

Comment:
"al"l it's not good name, some examples: attachments, attachmentList.
Also please check other places with uninformative names.

Reply:
I fixed all mistakes